load_agent.py

```diff
@@ -66,3 +66,6 @@ def episode(env, agent):
             print("Done!")
     print('test episode:', 'R:', R)
     agent.stop_episode()
+
+
+create_environment("../dataset-generator/image_locations.txt", "../dataset-generator/bounding_boxes.npy")
```

Comment:
Are you sure that this should be checked in? If yes, should it maybe be wrapped in the following way?

```suggestion
if __name__ == '__main__':
    create_environment("../dataset-generator/image_locations.txt", "../dataset-generator/bounding_boxes.npy")
```